certbot_dns_inwx/dns_inwx.py

```diff
@@ -204,13 +204,10 @@ def _find_domain(self, domain_name):
         for guess in domain_name_guesses:
             logging.debug('Testing {0} for domain {1}...'.format(guess, domain_name))
             try:
-                info = self.inwx.domain.info({'domain': guess})['resData']
+                # this raises a NameError when the domain does not exist (or any other problem occurred):
+                self.inwx.nameserver.info({'domain': guess})
             except:
                 continue
-            if not 'status' in info:
```

Comment:
There is no such status field in the response of nameserver.info, so I just dropped this check.